CONTRIBUTING.md

```diff
@@ -1,17 +1,25 @@
-Contributing
-============
+# Contributing
 
 Thanks for contributing!
 
 ## Before you contribute
 
-This package is a fork of [fast-deep-equal](https://github.com/epoberezkin/fast-deep-equal). This library has added handling for React. Before contributing, _please make sure the issue relates directly to this library and not fast-deep-equals_.
+This package is a fork of [fast-deep-equal](https://github.com/epoberezkin/fast-deep-equal). This library has added handling for React.
+Before contributing, _please make sure the issue relates directly to this library and not fast-deep-equal's_.
 
 We encourage pull requests concerning:
 
 * React features not handled in this library
-* Integrating updates from `fast-deep-equal`. This unfortunately, now requires more manual work to use the comment blocks in `index.js` to figure out what to paste and where.
-* Integrating tests from `fast-deep-equal`. This usually entails upgrading the `git`-based dependencies of `fast-deep-equal-git` and `npm`-published package of `fast-deep-equal` in `package.json:devDependencies`.
+* Integrating updates from `fast-deep-equal`
+
```

Comment:
is this just your prettier that adds the extra line breaks? i don't really care, i just want it to easily be consistent, and not a lot of dirty lines every time someone new changes the file.

Reply:
looks like these extra spaces actually make a visual difference:

![image](https://user-images.githubusercontent.com/2822048/80843323-3e655a00-8bc1-11ea-8f47-5743b39b2862.png)

Reply:
the 3 on the bottom, without extra line breaks, is what we want